### PR TITLE
Update jquery-validate.bootstrap-tooltip.js

### PR DESCRIPTION
**Pull request by luis226**
When the blur event happens on a control the tooltip flickers or sometimes dissapears, the problem its the line  'tooltip("destroy")' because when you destroy followed by tooltip create doesnt work, with this commit the tooltip message is changed without destroying it, solving the problem

### CODE REVIEW

**Comment by Thrilleratplay** [comment]
For your inputs, are you specifying "required" as a class or attribute?  Using the required attribute will, or should, leave the tooltips visible even on blur.  This is how the first example on the demo page behaves for me.  If it doesn't for you, can you provide the versions of Bootstrap, jQuery and jQuery Validation used as well as browser information?

**Comment by luis226** [comment]
Jquery v2.1.3
Bootstrap v3.3.2 
Jquery validate v1.13.1
Google Chrome v41.0.2272.89 m

Im using ASP MVC on serverside, it automatically renders the inputs with the needed attributes for jquery validation, the core of the problem its that i have more than one kind of validation for many inputs so lets say an input has required and number validation, when i fill the input the required message disappears and the "must be a number" message appears, when the lost focus event of the input triggers the tooltip flickers and dissapears even when the validation hasnt been satisfied and when the lostfocus event triggers again in the same input the tooltip reappears i started testing this on jsfiddle and it actually works all right, maybe the problem its that ASP.NET MVC uses some scripts on the middle like "jquery.unobtrusive-ajax.min.js" and "jquery.validate.unobtrusive.min" that are interfering with your boostraptooltipvalidate plugin, my modification fixed the problem for me, if you want give me your email so i can send you a MVC project example or the generated html so you can see the problem for yourself

**Comment by Thrilleratplay** [comment]
I have never worked with ASP.NET MVC and did not know it shipped with its own version of jQuery Validate.  I think you are on to something.  Take a look at [ jQuery Validation Unobtrusive  Native](http://johnnyreilly.github.io/jQuery.Validation.Unobtrusive.Native/index.html) and in particular [advantage #2's demo](http://johnnyreilly.github.io/jQuery.Validation.Unobtrusive.Native/AdvancedDemo/Tooltip.html).  If there is another project that handles a nonstandard version of jQuery Validate in a similar fashion, I am more inclined to point those developers to that project then modify this project and possibly introduce new bugs.

**Comment by luis226** [comment]
Nice, i didnt know there was a project that does the same for MVC scripts, thank you

**Comment by Thrilleratplay** [comment]
Thank you for pointing out there is a version of jQuery Validation that causes issues.  I added a note to the Readme for other ASP MVC developers.

**Comment by Thrilleratplay** [comment]
@luis226 I wanted to thank for this pull request.  It ultimately was added in a second version of this plugin to fix the flicker issue for Bootstrap 3.3.0+

**Comment by luis226** [comment]
No problem, its good to know that my pull request was useful, have a nice day

Date: Tue, 5 May 2015 05:11:41 -0700
